javascript/javascript3/week2/geolocation.js

```diff
@@ -0,0 +1,31 @@
+const setTimeoutPromise = (timeout) =>{
```

Comment:
Suppose I want to do something like this:

```
(async () => {
  await promFunction(8);
  console.log("message after 8 seconds")
})()
```

How could that be accomplished?